website/docs/r/cos_bucket.html.markdown

```diff
@@ -128,8 +128,28 @@ resource "ibm_cos_bucket" "archive_rule_cos"{
     days = 0
     type = "GLACIER"
   }
+  expire_rule{ 
```

Comment:
Might be worth running the example code through `terraform fmt` as spacing is all over the place.